webf/lib/src/svg/svg_render_object_builder.dart

```diff
@@ -73,7 +73,9 @@ class SVGRenderBoxBuilder {
             .toDartString(length: element.original_tag.length)
             .toUpperCase();
         final renderBox = getSVGRenderBox(tagName);
-
+        if (renderBox == null) {
+          return false;
```

Comment:
Is it more suitable to return a null value when the renderBox is null, as the expected value of visitSVGTree is RenderBoxModel ?

Reply:
It shouldn't be necessary. You can take a look at this complete code.